lib/puppet_x/bsd/hostname_if.rb

```diff
@@ -192,22 +282,30 @@ def lines
 
         # please_help_add_support_for = [
         #   'mpe',
+        #   'mpw',
         #   'ppp',
         #   'pppoe',
         #   'sl',
+        #   'svlan',
+        #   'vxlan',
         # ]
 
+        # Process IP address configuration
+        process_addresses(@addresses) {|line|
```

Comment:
Since the subclasses are using `PuppetX::BSD::Hostname_if::Inet`, what do you think about using that here in place of having two sets of code to process addresses?